adapters/defra.js

```diff
@@ -438,6 +438,9 @@ let metadata = {
   'Belfast Stockman\'s Lane':
    { city: 'Belfast',
      coordinates: { latitude: 54.572586, longitude: -5.974944 } },
+  'Derby St Alkmund\'s Way':
+   { city: 'Bass\'s Recreation Ground St Alkmund\'s Way Derby',
```

Comment:
They were taken from the results of `defra-locations.js` script. I quickly assumed that the results were used as is. I'll update with their city names: Derby and Plymouth respectively.